extensions/wikia/AdEngine/js/slot/bottomLeaderboard.js

```diff
@@ -26,11 +39,35 @@ define('ext.wikia.adEngine.slot.bottomLeaderboard', [
 			if (!pushed && pushPos < scrollPosition) {
 				pushed = true;
 				doc.removeEventListener('scroll', pushSlot);
-				win.adslots2.push(slotName);
-				log(['pushSlot', 'Pushed ' + slotName], 'debug', logGroup);
+
+				if (!collapseOnSameViewport()) {
+					win.adslots2.push(slotName);
+					log(['pushSlot', 'Pushed ' + slotName], 'debug', logGroup);
+				}
 			}
 		});
 
+	function collapseOnSameViewport() {
+		if (adContext.get('opts.isBLBViewportEnabled') && isUap() && viewportObserver.sameViewport(slotName, breakSlots)) {
+			eventDispatcher.dispatch('adengine.slot.status', {
+				slot: {
+					name: slotName
+				},
+				status: 'viewport',
```

Comment:
We already have a solution to collapse slots with specific type:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/btfBlocker.js#L73
and this flow makes clear tracking